lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java

```diff
@@ -245,7 +246,7 @@ private void drawTextWithFont(
     String text = documentData.text;
     TextDelegate textDelegate = lottieDrawable.getTextDelegate();
     if (textDelegate != null) {
-      text = textDelegate.getTextInternal(text);
+      text = textDelegate.getTextInternal(getName(), text);
```

Comment:
This isn't a key path name. This is the layerName. Key path would be the fully qualified list of elements up to the root composition

Reply:
I see that, I if I am understanding this is similar to iOS though looking at the implementation. It is only the layer name.

Reply:
For more context, this is what I am referring to.

https://github.com/airbnb/lottie-ios/blob/64564c12ba0c258f5a5d9b1e81455b1f1da2315d/lottie-swift/src/Private/LayerContainers/CompLayers/CompositionLayer.swift#L63

    CompositionLayer {
      init(layer: LayerModel, size: CGSize) {
          ...
          self.keypathName = layer.name
          ...
      }
    }

I didn't think this was the true keypath either, but just a string representing the layer name.

Reply:
@greggiacovelli Interesting. It does seem like iOS is using the name slightly inconsistently. For the same of consistency within the Android library, I'd prefer `layerName` here

Reply:
Ok will do. Thanks for the guidance and I will append the PR in a bit

Reply:
addressed in 2099504f